server/dummyControllers/requestController.js

```diff
@@ -0,0 +1,40 @@
+import Request from './../dummyModels/request';
+import User from './../dummyModels/user';
+
+/**
+ * it is a class that control all request api;
+ */
+class requestController {
+  /**
+     * it GET all requests
+     * @param {string} req
+     * @param {string} res
+     * @returns {object} object
+     */
+  static getAllRequests(req, res) {
+    const admin = User.filter(user => user.role === 'admin');
+    // check if user is admin to get all request
+    if (!admin) {
+      return res.status(403).json({
+        error: true,
+        message: 'you dont have the priviledge for this request'
+      });
+    }
+    // get all request
+    if (Request) {
+      return res.status(200).json({
+        error: false,
+        message: 'success',
+        Request
+      });
+    }
+    // return an error if request not found
+    return res.status(404).json({
```

Comment:
When writing code been explicit will save you time in the long run...use an else statement to be more explicit

Reply:
ok. 
I will be more explicit